solutions/devsprint-adriano-song-1/RealEstateAppTests/LoginViewSnapshotTest.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  LoginViewSnapshotTest.swift
+//  RealEstateAppTests
+//
+//  Created by Fabio Martinez on 17/02/22.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import RealEstateApp
+
+class LoginViewSnapshotTest: XCTestCase {
+
+    func testLoginView() throws {
+        let loginView = LoginView()
+        assertSnapshot(matching: loginView, as: Snapshotting.image(size: CGSize(width: 300, height: 350)), record: false)
```

Comment:
your test was failed, look into the library documentation here: (https://github.com/pointfreeco/swift-snapshot-testing) always try to avoid hardcoded screen size like this `Snapshotting.image(size: CGSize(width: 300, height: 350)`, try always to match a ViewControlller not a view and also use default simulator configuration like `assertSnapshot(matching: vc, as: .image(on: .iPhoneSe))`